core/src/main/resources/db/migration/V1__initial_schema.sql

```diff
@@ -43,14 +45,14 @@ CREATE TABLE UpdateSpec (
 );
 
 CREATE TABLE RequiredPackage (
-  update_request_id CHAR(36) NOT NULL,
-  vin varchar(64) NOT NULL,
-  package_name VARCHAR(200) NOT NULL,
-  package_version VARCHAR(200) NOT NULL,
+    update_request_id CHAR(36) NOT NULL,
```

Comment:
It makes it consistent with the rest of the file, these lines are the only ones with 2-space indents. I'm happy to remove them though if you think it's minor.